app/views/pages/conditions/new.html.erb

```diff
@@ -9,18 +9,13 @@
       <%= t("page_titles.routing_page") %>
     </h1>
 
-    <!-- If the question "where were you born" Change -->
-    <dl class="govuk-summary-list">
-      <div class="govuk-summary-list__row">
-        <dt class="govuk-summary-list__key"><%= t("new_condition.routing_page_text") %></dt>
-        <dd class="govuk-summary-list__value">“<%= page.question_text %>”</dd>
-        <dd class="govuk-summary-list__actions">
-          <%= govuk_link_to routing_page_path(form) do %>
-            <%= t("new_condition.routing_page_change_html") %>
-          <% end %>
-        </dd>
-      </div>
-    </dl>
+    <%= govuk_summary_list do |summary_list|
+      summary_list.with_row do |row|
+        row.with_key { t("new_condition.routing_page_text") }
+        row.with_value { page.question_text }
+        row.with_action(text: "Change", href: routing_page_path(form), visually_hidden_text: "routing question")
```

Comment:
Very minor but wonder if we should have a translation for this 'Change' text?

Reply:
I think we use `Edit` everywhere else so wonder if this is another question for the designers. Can probably ignore in this PR